letter-shop/index.md

```diff
@@ -0,0 +1,634 @@
+
+> **NOTE:** As we go through this tutorial we'll be using different functions of the Ledger Plugin Interface. You can find the reference documentation in [IL-RFC 0004](https://interledger.org/rfcs/0004-ledger-plugin-interface/).
+
+We've put all of the plugin config into a single file called `plugins.js`. The default plugin we use is the *XRP Escrow* plugin which allows us to connect to the XRP Testnet Ledger.
+
+The *XRP Escrow* plugin is a wrapper around [RippleLib](https://github.com/ripple/ripple-lib), and exposes the Ledger Plugin Interface (LPI). In the code, you see that an 'ilp-plugin-xrp-escrow' Plugin is being configured with secret, account, server, and prefix. The first three values come from the [XRP Testnet Faucet](https://ripple.com/build/xrp-test-net/).
+
+Edit this file in your favourite text editor and follow the instructions in the code comments to:
+
+1. Get different XRP Testnet credentials for both the shop and the customer.
+1. Configure the XRP Ledger plugins with those new credentials
+
+> **IMPORTANT:** Even if you are using the completed tutorial code you'll need to set this up so you are using your own test accounts.
+
+Now try running your server again.
+
+```shell
+$ node shop.js
+```
+
+You should see something like:
+
+```
+== Starting the shop server ==
+```
+
+Inside `shop.js` you'll see we got an instance of the ledger plugin you configured in `plugins.js`:
+
+```js
+const plugin = require('./plugins.js').xrp.Shop()
+```
+
+Now we need to **do something** with that plugin. Replace the text `// Do something...` with the following:
+
+```js
+console.log(` 1. Connecting to an account to accept payments...`)
+
+plugin.connect().then(function () {
+  
+  //Get ledger and account information from the plugin
+  const ledgerInfo = plugin.getInfo()
+  const account = plugin.getAccount()
+
+  console.log(`    - Connected to ledger: ${ledgerInfo.prefix}`)
+  console.log(`    -- Account: ${account}`)
+  console.log(`    -- Currency: ${ledgerInfo.currencyCode}`)
+  console.log(`    -- CurrencyScale: ${ledgerInfo.currencyScale}`)
+
+  //Covert our cost (10) into the right format given the ledger scale
+  const normalizedCost = cost / Math.pow(10, parseInt(ledgerInfo.currencyScale))
+
+  // Handle incoming web requests...
+
+  // Handle incoming transfers...
+
+})
+```
+
+Stop the server and restart it with the new code. Now it will use the ledger plugin to connect to the XRP Testnet Ledger. When it is connected (usually a few seconds) you'll see some logging in the console providing info about the ledger.
+
+We can get a lot of info from the plugin about the ledger and the account it is connected to using the `getInfo()` and `getAccount()` methods.
+
+> **NOTE:** The currency of the ledger is XRP and the currency scale is 6. That means that to send 1 XRP to this ledger we must send an Interledger payment with an amount of 1000000 (1 million Drops).
```

Comment:
Great addition! This is something every newcomer will ask, and I forgot to talk about it.